libraries/tests/rtos/mbed/mutex/main.cpp

```diff
@@ -16,8 +16,7 @@
     #define STACK_SIZE DEFAULT_STACK_SIZE
 #endif
 
-void print_char(char c = '*')
-{
```

Comment:
why are you changing the style? It is not the one which is mostly used in files here